### ISSUE TITLE
Unknown Error After Reload

### ISSUE THREAD

**Issue by LDexter** [LuaLS/vscode-lua#86]
### Which OS are you using?

MacOS Ventura 13.0, Intel chip

### Expected Behaviour

No error message after reloading for extension update

### Actual Behaviour

Error message after reloading for extension update

I also see the following error:

```
[2023-03-03 17:32:38] |   ERROR   |         Git        | fatal: Not a git repository (or any of the parent directories): .git

```

### Reproduction steps

1. Go to 'reload after ext update'
2. Click 'error message'
3. See error 'as pasted into log'

### Additional Notes

Not really sure what went wrong. Putting my faith in the log paste for this one.

### Log

<details><summary>Retrieved Log</summary>

```
#### STARTUP ####
[2023-03-03 17:32:35] |   DEBUG   |     Filesystem     | Created directory at "/Users/levimiller/Library/Application Support/Code/User/globalStorage/sumneko.lua/addonManager"
[2023-03-03 17:32:38] |   DEBUG   |         Git        | Cloned LLS-Addons
[2023-03-03 17:32:38] |   ERROR   |         Git        | fatal: Not a git repository (or any of the parent directories): .git


```

</details>

**Comment by carsakiller** [comment]
Hello! There was an issue with file paths on UNIX machines that should now be resolved in `3.6.16`. Could you please confirm it is now working?

**Comment by carsakiller** [comment]
Please follow this issue in https://github.com/LuaLS/vscode-lua/issues/81
